src/backend/distributed/utils/node_metadata.c

```diff
@@ -469,20 +469,26 @@ master_update_node(PG_FUNCTION_ARGS)
 
 	char *newNodeNameString = text_to_cstring(newNodeName);
 
+	List *placementList;
```

Comment:
we usually initialise all variables (to NIL)

Reply:
done